interface/themes/style_tan.css

```diff
@@ -673,7 +673,7 @@ tr.odd {
 }
 /* set the text color inside button to white when it is visisted or hovered */
 .css_button:visited{
-	color:white;
+	color:brown;
```

Comment:
great you did this in the theme file.

Reply:
LGTM . I hope the new UI fixes all these